### DISCUSSION TITLE
犯罪現場燒腦的事就交給AI

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#698] [General]
在犯罪現場這樣的3D空間中，能否以噴濺的血跡推測出嫌犯犯案時的位置呢？AI辦得到！
嫌犯砍殺被害人時，動作很大，血會隨著犯案工具噴濺出去，拋甩在四面牆上及天花板，透過AI計算這些血點的角度，可模擬出血源的來源線，發現交會在同一位置，可推算兇嫌犯案的正確位置。像這樣的犯案現場，以前的時代須以人工的方式判斷，還需寫報告以書面方式呈現。現在能透過AI快速計算呈現，除了節省時間，也節省納稅人的錢（因為能減少DNA的重複取樣及採證費用）。
除此之外有很多犯罪現場的細節，AI都能協助警方找到蛛絲馬跡。

AI其實很多時候都像個小幫手，除了能協助警方偵辦案件，也能節省一些不必要的社會成本。在各個產業裡，若能好好利用AI，那肯定會事半功倍。

文章來源：https://www.thenewslens.com/feature/aishow/175796

**Comment by titan92601** [comment]
沒想到AI竟然可以做到這種程度。我覺得可能之後再進步點連模擬事故發生都可以靠AI進行投影還原，不但可以加速破案也少了許多人力資源在這上面。

**Comment by HsuChen88** [comment]
柯南要失業了，人們也該收起犯罪的想法了。

**Comment by Vincent20030314** [comment]
感覺這種技術還要一段時間才能實現，畢竟人類都不一定能準確判斷。
